pom.xml

```diff
@@ -335,7 +566,7 @@
                         <sourceDirectory>src/main/java</sourceDirectory>
                     </sourceDirectories>
                     <configLocation>${maven.multiModuleProjectDirectory}/config/checkstyle/checkstyle.xml</configLocation>
-                    <propertyExpansion>config_loc=${maven.multiModuleProjectDirectory}/config/checkstyle</propertyExpansion>
+                    <propertyExpansion>config_loc=${maven.multiModuleProjectDirectory}/config/checkstyle</propertyExpansion> <!-- Needs to be synchronized with the caching config for checkstyle in the Develocity Caching Configuration (gradleEnterprise config). -->
```

Comment:
I don't understand the comment.

Reply:
The path needs to be the same as the checkstyle caching setup (line 308 currently).

That's because of two things: 
1. `maven.multiModuleProjectDirectory` returns an absolute path which breaks relocatibility
2. This is just a string, and the cache key depends on file contents, so it needs to be set up as a file input